bahmni/src/main/java/org/avni_integration_service/bahmni/contract/OpenMRSSavePersonAddress.java

```diff
@@ -0,0 +1,46 @@
+package org.avni_integration_service.bahmni.contract;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class OpenMRSSavePersonAddress {
```

Comment:
Do you need this? Isn't there already a class called "OpenMRSPersonAddress"? .. whats specific about this class?

Reply:
For consistency we added this class. Avni is maintaining separate classes to sync person details